packages/@react-spectrum/table/src/TableViewBase.tsx

```diff
@@ -649,7 +648,6 @@ function TableVirtualizer(props) {
             role="presentation"
             className={classNames(styles, 'spectrum-Table-headWrapper')}
             style={{
-              width: visibleRect.width,
```

Comment:
Looks like this width wasn't actually needed? This simplifies the logic greatly since the table header wrapper lets the browser determine its width via its flex container width's, matching the scroll body behavior.

Reply:
🤞

Reply:
🤞🤞